RELEASING.md

```diff
@@ -0,0 +1,14 @@
+# Publishing new crate versions
+New versions of both the `mongocrypt-sys` and `mongocrypt` crates can be released with the `publish.sh` script; they do not have to be released in lockstep, although API changes in `mongocrypt-sys` will probably require changes in `mongocrypt` as well.
+
+1. When publishing the `mongocrypt` crate, push a change updating the `mongocrypt-sys` and `bson` dependencies to the most recent published versions.
+1. Create a tag combining the crate name and version to be published, e.g. `mongocrypt-sys-0.1`.  If pushing both crates, create two tags (they can point to the same commit).
+1. Push the tag(s) upstream.
+1. Run the publish script with the `TAG`, `TOKEN`, and `CRATE` variables:
+
+        TAG=0.1 \
```

Comment:
nit: can we name this `VERSION` instead? it's a little unclear that this should just be the version part of the tag rather than the whole thing. it might also be helpful to use placeholders for `TAG` and `CRATE` to make it more obvious they need to be updated, e.g.

```
VERSION=<version to be published> \
..
CRATE=<mongocrypt | mongocrypt-sys> \
```

Reply:
Done.